### DISCUSSION TITLE
AI Smart Assistance via IBM's watsonx

### DISCUSSION THREAD

**Discussion by lenwood** [dbeaver/dbeaver#23418] [Q&A]
I'd like to take advantage of AI smart assistance. My company only allows usage of IBM's watsonx, which currently isn't an option. Is there a way that I could enable the [watsonx API](https://www.ibm.com/docs/en/watsonx-as-a-service?topic=started-watsonx-apis)?

**Comment by LonwoLonwo** [comment]
Hello @lenwood

We do not support watsonx for now. You can create a feature request ticket for it here: https://github.com/dbeaver/dbeaver/issues
